packages/checkout-com/nuxt/index.js

```diff
@@ -1,13 +1,39 @@
 import path from 'path';
 
+const isScriptInArray = (headEntries, scriptUrl) => headEntries.some(entry => entry.src === scriptUrl);
+
+const defaultPaymentMethods = {
+  cc: true,
+  paypal: true,
+  klarna: false
+};
+
+const paymentMethodSdk = {
+  cc: 'https://cdn.checkout.com/js/framesv2.min.js',
+  paypal: null,
+  klarna: 'https://x.klarnacdn.net/kp/lib/v1/api.js'
+};
+
 export default function CheckoutComModule(moduleOptions) {
+  const scripts = this.options.head.script;
   this.addPlugin({
     src: path.resolve(__dirname, './plugin.js'),
-    options: {
-      publicKey: moduleOptions.publicKey
-    }
+    options: moduleOptions
   });
 
+  const paymentMethods = {
+    ...defaultPaymentMethods,
+    ...(moduleOptions.paymentMethods ? moduleOptions.paymentMethods : {})
+  };
+
+  for (const [paymentMethod, enabled] of Object.entries(paymentMethods)) {
+    if (enabled && paymentMethodSdk[paymentMethod] && !isScriptInArray(scripts, paymentMethodSdk[paymentMethod])) {
+      scripts.push({
+        src: paymentMethodSdk[paymentMethod]
+      });
```

Comment:
Shouldn't this have at least `async` attribute?

Reply:
Sure it could